api/api/src/test/scala/uk/ac/wellcome/platform/api/elasticsearch/VarientQueriesTest.scala

```diff
@@ -0,0 +1,158 @@
+package uk.ac.wellcome.platform.api.elasticsearch
+
+import com.sksamuel.elastic4s.ElasticDsl.search
+import com.sksamuel.elastic4s.Index
+import org.scalatest.{FunSpec, Matchers}
+import org.scalatest.concurrent.ScalaFutures
+import uk.ac.wellcome.elasticsearch.test.fixtures.ElasticsearchFixtures
+import uk.ac.wellcome.models.work.generators.{
+  ContributorGenerators,
+  GenreGenerators,
+  SubjectGenerators,
+  WorksGenerators
+}
+import uk.ac.wellcome.platform.api.generators.SearchOptionsGenerators
+import uk.ac.wellcome.platform.api.services.ElasticsearchService
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.util.Random
+import com.sksamuel.elastic4s.ElasticDsl._
+import com.sksamuel.elastic4s.requests.searches.sort.SortOrder
+
+class VarientQueriesTest
```

Comment:
`s/Varient/Variant`, might be clearer as `QueryVariantsTest` if I've understood its purpose right